.gitignore

```diff
@@ -1,7 +1,3 @@
 .idea
 .env
-/bin/
-/pkg/
-/src/github.com/
-/src/golang.org/
-/src/gopkg.in/
+vendor/
```

Comment:
If you use a vendor directory, you have to commit it and push in your git repository as well. But I suggest you don't use vendors

Reply:
Should we commit only pure vendor directory with gitkeep? Without dependcies, right?
Why it's better to don't use vendors,  is it not welcome in go world?